packages/e2e-tests/cypress/integration/full-planning/before-you-start/granted-or-refused-application/granted-or-refused-application-steps.js

```diff
@@ -13,7 +13,7 @@ const pageTitle = 'Was your planning application granted or refused? - Before yo
 const pageHeading = 'Was your planning application granted or refused?';
 const url = '/before-you-start/granted-or-refused';
 const decisionDatePageUrl = '/before-you-start/decision-date';
-const decisionDateDuePageUrl = '/before-you-start/decision-date-due';
```

Comment:
Just checking this isn't the decision date page, as that url is different: https://pins-ds.atlassian.net/browse/AS-3673?atlOrigin=eyJpIjoiOWRmODU5NmE3Y2Y0NGU3Njk1YzI5NmJiYWNhY2Q4MjEiLCJwIjoiaiJ9

Reply:
No, this is not _decision date_ page that you worked on